geps/gep-1619/index.md

```diff
@@ -49,13 +61,26 @@ session affinity, as this design is expected to be addressed within a separate G
 Session persistence is when a client request is directed to the same backend server for the duration of a "session". It is achieved when a client directly provides information, such as a header, that a proxy uses as a reference to direct traffic to a specific server. Persistence is an exception to load balancing: a persistent client request bypasses the proxy's load balancing algorithm, going directly to a backend server it has previously established a session with.
 
 Session persistence enables more efficient application workflows:
+
 1. Better performance: Maintaining a single session allows a server to cache information about a client locally reducing the need for servers to exchange session data and overall storage needs.
 2. Seamless client experience: Clients can reconnect to the same server without re-authenticating or re-entering their information.
 
 Some of the concerns of session persistence are the duration and expiration of the session, security of the transaction stream, and storage of the context or state.
 
 Session affinity, not to be confused with session persistence, uses an existing attribute of the request to consistently send to the same backend. Session affinity can be considered a weaker form of session persistence: it is not guaranteed to persist a connection to the same backend server if certain attributes of the request or the backends are changed.
 
+#### A Session vs. Session Persistence
+
```

Comment:
This section is quite confusing ( at least to me ).

Are you trying to say that the 'user session' concept - 'login', 'do stuff', 'logout' is broader and during the user session multiple services will use the 'session id' with either storage-based and 'stickiness' implementations ?

"Session data must be accessbile to every backend" could be explained maybe as "a user session spans multiple subsystems, and can be implemented using a combination of storage - using session ID and user as key, replicating session data  and 'sticky' backends'

If that's what you're talking about - it's useful to emphasize the 'user session' spans multiple different subsystems and backends. But we don't do a very good job in this API to capture this use case, the current API is focused on a single backend in isolation.

Reply:
> Are you trying to say that the 'user session' concept - 'login', 'do stuff', 'logout' is broader and during the user session multiple services will use the 'session id' with either storage-based and 'stickiness' implementations ?

No, not intentionally. The goal for this paragraph was just to define what a "session" (or "user serssion" as you mentioned) is. I agree the current GEP is focused on a single backend in isolation, so I don't want to start something new regarding multiple subsystem sessions.

I'll save this commit/change for a future PR (I added it a bit on a whim anyways) and we can discuss it there.